packages/css/CODE_CONVENTION.md

```diff
@@ -17,6 +18,13 @@
   - Avoid reassigning
   - Use the word `spacing` when we use it to do more than just set a property. For example in `padding`, but also in a calc for `margin-inline`
   - States should be in this pattern: `--ds-{COMPONENT}-{PROPERTY}--{STATE}`
+  - `--_dsc` for internal
+- Prefix keyframes with `ds-{COMPONENT}`
+- Use logical block and inline properties
+  - Use `margin-block` and `margin-inline` instead of `margin-top`, `margin-bottom`, `margin-left`, and `margin-right`
+  - Use `padding-block` and `padding-inline` instead of `padding-top`, `padding-bottom`, `padding-left`, and `padding-right`
+  - Use `border-block-start` and `border-block-end` instead of `border-top` and `border-bottom`
+  - Use `border-inline-start` and `border-inline-end` instead of `border-left` and `border-right`
```

Comment:
the complete list of logical properties is quite exhaustive but perhaps it coul be linked to https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values